### DISCUSSION TITLE
Adding and removing fonts

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#937] [General]
I have no problem adding custom fonts. But now I want to remove all the standard fonts and in general set the BODY font to one of my custom fonts by default.

**Comment by meetmandeep** [comment]
You could set the fonts for Text, Headings, Buttons, or Menu from Menu > Design > Themes > Basic > Settings > Site

At this time, there is no option to remove the standard web fonts. There might be a workaround but it would be overwritten with every Vanjaro Update.
